web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -130,6 +130,9 @@
                   'formatters/xsl-view?root=div&view=advanced'
             }]
           },
+          'grid': {
+            'related': ['parent', 'children', 'services', 'datasets']
+          },
```

Comment:
Fixed in https://github.com/geonetwork/core-geonetwork/commit/d37b1079fd4d4d85030dd6fec5bf19ae1e157369